build/tester.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+image="quay.io/helmpack/chart-testing"
+version="v2.4.1"
+
+if [ "${1-}" = "pull" ]; then
+  docker pull "${image}:${version}"
+  exit 0
+fi
+
+if [ "${1-}" = "version" ]; then
+  echo "${version}"
+  exit 0
+fi
+
+if [ -t 0 ]; then
+  tty=--tty
+fi
+
+# Absolute path to the toplevel helm-charts directory.
+helm_charts_toplevel=$(dirname "$(cd "$(dirname "${0}")"; pwd)")
+
+container_home=/charts/
+
+# Run our tester container with a set of volumes mounted that will
+# allow the container to store persistent data on the host computer.
+
+vols=
+vols="${vols} --volume=${helm_charts_toplevel}:${container_home}"
```

Comment:
Why not make this a single line?

`vols=--volume=${helm_charts_toplevel}:${container_home}"`

Reply:
That was vestigial from the script I used as a template for this one and I hadn't changed it. Updated to just one line.